src/components/NewDesign/Modals/SelectModals/SelectTokenModal/SelectTokenModal.tsx

```diff
@@ -4,10 +4,9 @@ import CustomScrollbar from '../CustomScrollbar'
 import icons from '@static/icons'
 import useStyles from '../style'
 import searchIcon from '@static/svg/lupa.svg'
-import { BN } from '@project-serum/anchor'
 export interface ISelectTokenModal {
-  tokens: Array<{ symbol: string; balance?: BN; decimals?: number }>
-  commonTokens: Array<{ symbol: string; balance?: BN; decimals?: number }>
+  tokens: Array<{ symbol: string, name: string, icon: string }> | null
+  commonTokens: Array<{ symbol: string, name: string, icon: string }>
```

Comment:
if tokens list might be null, common tokens list might be as well

Reply:
okay